jbrowse/resources/external/minimalSession.json

```diff
@@ -30,6 +30,12 @@
     }
   },
   "location": "1:196,734,341..197,535,702",
+  "displays": [
+    { "name": "Predicted Function",
+      "properties": ["AF_ESP", "ALLELEID"]},
+    { "name": "Regulatory Data",
+      "properties": ["CLNDN", "CLNVC"]}
+  ],
   "tracks": [
     {
       "type": "VariantTrack",
```

Comment:
see comment below about renaming the adapter to make it unique. if we do that, we'd want to have this track opt-in to our custom adapter, which i think is a better overall design.

Reply:
Added issue #48 , will address in another PR